core/input/input_event.h

```diff
@@ -64,6 +64,7 @@ class InputEvent : public Resource {
 public:
 	static const int DEVICE_ID_EMULATION;
 	static const int DEVICE_ID_INTERNAL;
+	static const int DEVICE_ID_ALL_DEVICES;
```

Comment:
These could be inlined.

Reply:
How would the change to an enum affect the GDScript-API?

https://github.com/godotengine/godot/blob/44fa552343722bb048e2d7c6d3661174a95a8a3c/core/input/input_event.cpp#L136

Reply:
It should make no difference I think.
It could stay as constants, but enum is better organization-wise, as the values are related.

Reply:
I believe, we are currently stuck with inline constants, if we don't want to break compatibility.

Changing the constants to enums and running the testsuite causes the following error:
```
./tests/core/object/test_class_db.h:880:
TEST SUITE: [ClassDB]
TEST CASE:  [ClassDB] Add exposed classes, builtin types, and global enums

./tests/core/object/test_class_db.h:810: FATAL ERROR: REQUIRE_FALSE( constant_name.contains("::") ) is NOT correct!
  values: REQUIRE_FALSE( true )
  logged: Constant contains '::', check bindings to remove the scope: 'InputEvent.DeviceId::EMULATION'.
```